i2c.go

```diff
@@ -3,7 +3,7 @@ package drivers
 // I2C represents an I2C bus. It is notably implemented by the
 // machine.I2C type.
 type I2C interface {
-	ReadRegister(addr uint8, r uint8, buf []byte) error
-	WriteRegister(addr uint8, r uint8, buf []byte) error
+	// ReadRegister(addr uint8, r uint8, buf []byte) error
+	// WriteRegister(addr uint8, r uint8, buf []byte) error
```

Comment:
Why not remove them entirely? I see no point in keeping them but commenting them out.

Reply:
You got me- I have this horrible tendency to comment things out and just.... leave them there. Will remove!